torchci/pages/tests.tsx

```diff
@@ -183,19 +183,19 @@ export default function GatherTestsInfo() {
 
       <Grid container spacing={4}>
         <GenerateTestInsightsOverviewTable
-          workflowName={"trunk"}
```

Comment:
I was leaving `pull` this at the bottom without realizing it, moving it up the stack where it should be